injected/src/features/broker-protection/captcha-services/providers/registry.js

```diff
@@ -0,0 +1,27 @@
+import { CaptchaFactory } from '../factory';
+import { ReCaptchaProvider } from './recaptcha';
+import { CloudFlareTurnstileProvider } from './cloudflare-turnstile';
+import { HCaptchaProvider } from './hcaptcha';
+
+const captchaFactory = new CaptchaFactory();
+
+captchaFactory.registerProvider(
+    new ReCaptchaProvider({
+        type: 'recaptcha2',
+        providerUrl: 'https://www.google.com/recaptcha/api2',
+        responseElementName: 'g-recaptcha-response',
+    }),
+);
+
+captchaFactory.registerProvider(
+    new ReCaptchaProvider({
+        type: 'recaptchaEnterprise',
+        providerUrl: 'https://www.google.com/recaptcha/enterprise',
+        responseElementName: 'g-recaptcha-response',
+    }),
+);
+
+captchaFactory.registerProvider(new CloudFlareTurnstileProvider());
```

Comment:
For this pass, let's not register cloudFlare or hCaptcha until we test explicitly, just to reduce the surface area.